rs/cli/src/commands/version/revise/guest_os.rs

```diff
@@ -33,8 +33,8 @@ impl ExecutableCommand for GuestOs {
                 &ic_management_types::Artifact::GuestOs,
                 &self.version,
                 &self.release_tag,
-                self.force,
-                ctx.forum_post_link(),
+                self.ignore_missing_urls,
+                ctx.forum_post_link().expect("Forum post link is required for this proposal"),
```

Comment:
Can this go in `validate` step for this Command? It will then use clap abstractions for exiting

Reply:
I wanted to have that, yes. But validate does not have access to the command, only to ctx, and the refactor to get the into the validate isn't quite straightforward. But I do feel we should make that change, so I agree with you.

Reply:
fixed, PTAL